locations/spiders/espressolab.py

```diff
@@ -0,0 +1,58 @@
+import re
+from typing import Iterable
+
+import scrapy
+from lxml import etree
+from scrapy import http
+
+from locations.categories import Categories, apply_category
+from locations.pipelines.address_clean_up import clean_address
+
+STORES_PAGE_URL = "https://espressolab.com/subeler/"
+WEBSITE_ROOT = "https://espressolab.com"
+STORE_LINKS_XPATH = "/html/body/form/div[4]/div[4]/div[1]/div/div/div/div[1]/div/div/div[2]/ul/li/a/@href"
+STORE_NAME_XPATH = "//h1/text()"
+STORE_ADDRESS_XPATH = "//div[contains(@class, 'address')]"
+MAP_SCRIPT_XPATH = "/html/head/script[7]/text()"
+MAP_SCRIPT_REGEX = re.compile(r"google\.maps\.LatLng\([ ]*([-0-9.]*)[ ]*,[ ]*([-0-9.]*)[ ]*\);")
```

Comment:
No need to pull these all out into constants. It makes it harder to read the code below.

Reply:
I've put these into the code below in b073d8fb73606c1e29d75b3ab85a239158859e36.